### DISCUSSION TITLE
Atoum 4.1

### DISCUSSION THREAD

**Discussion by cedric-anne** [atoum/atoum#885] [General]
PHP `8.2` will be released at the end of current month. As current `master` branch already contains compatibility fixes for this PHP version, I propose to release the `atoum/atoum` `4.1` version ASAP.

**Comment by Grummfy** [comment]
hello,
indeed
I'm not really available right now (sick) but I have created a todo list https://github.com/atoum/atoum/issues/886 if you want

Otherwise it will probably be there before end of the month

**Reply by cedric-anne** [reply]
Thanks a lot for the release!

**Reply by Grummfy** [reply]
thanks to you, I should have more time to work on it, next year
